airbyte-integrations/connectors/source-vitally/manifest.yaml

```diff
@@ -219,13 +229,17 @@ definitions:
             type: CursorPagination
             page_size: 100
             cursor_value: "{{ response.next }}"
+        decoder:
+          type: JsonDecoder
       schema_loader:
         type: InlineSchemaLoader
         schema:
           $ref: "#/schemas/users"
   base_requester:
     type: HttpRequester
-    url_base: "{{ 'https://' + config['domain'] + '.' if config.domain else ''  + 'rest.vitally.io/resources' }}"
+    url_base: >-
+      {{ 'https://' + config['domain'] if config.domain else 'rest.vitally.io'
+      }}/resources
```

Comment:
This doesn't match the description below for the custom domain. Can you explain what are you fixing in custom domain?

Reply:
I don't like the original wording, it's a bit confusing.

Vitally's API is either e.g.:

https://mycustomdomain.rest.vitally.io/resources/accounts/

or:

https://rest.vitally.io/resources/accounts/

The user needs to provide `mycustomdomain.rest.vitally.io`

Examples:

https://rest.vitally.io/resources/accounts/ -> no custom domain
https://uptick.rest.vitally.io/resources/accounts/ -> custom domain

Example showing the old setup is wrong:

https://uptick.rest.vitally.io/accounts/ (404 not missing auth)

Reply:
> I don't like the original wording, it's a bit confusing.

But you keep the original description and change the behavior. Your current proposal change is not correct. The existing connector request to provide only the `subdomain` but the parameter is called `domain` which can be confusing but the example is clear. Either changes will result in a breaking change to existing users.